src/Adapters/Logger/WinstonLoggerAdapter.js

```diff
@@ -60,6 +60,22 @@ export class WinstonLoggerAdapter extends LoggerAdapter {
     return logger.error.apply(undefined, arguments);
   }
 
+  warn() {
+    return logger.warn.apply(undefined, arguments);
+  }
+
+  verbose() {
+    return logger.verbose.apply(undefined, arguments);
+  }
+
+  log() {
```

Comment:
https://github.com/ParsePlatform/parse-server/blob/fa736f1df7b6a4f6dbc3f08b2f9db8711de23694/src/ParseServer.js#L181

configured for silly?

I'd suggest we just implement the full sweet?  debug is useful.  https://github.com/winstonjs/winston#logging-levels